nginx/default.toml

```diff
@@ -34,5 +40,18 @@ tcp_nodelay = "on"
 # http.keepalive_timeout: Timeout on client connection keepalive, in seconds. Default = 75
 keepalive_timeout = 60
 
+# http.server_names_hash_bucket_size: Sets the bucket size for the server names hash tables.
+# The default value depends on the size of the processor’s cache line. Default = 32
+server_names_hash_bucket_size = 64
+
+www-data = "/hab/svc/nginx/data"
+global_ssl_password_file = "# ssl_password_file /home/hab/.ssh/tls/global.pass;"
```

Comment:
This is a misnomer, since you're supplying a commented out directive with a filename, not only a file.